### ISSUE TITLE
Slurm Executor ignores additional_setup_lines

### ISSUE THREAD

**Issue by tjader** [sampsyo/clusterfutures#4]
Due to a misplaced right-parenthesis.
```
    def _start(self, workerid, additional_setup_lines):
        return slurm.submit(
            '{} -m cfut.remote {}'.format(sys.executable, workerid, additional_setup_lines=additional_setup_lines)
        )
```
Should be
```
    def _start(self, workerid, additional_setup_lines):
        return slurm.submit(
            '{} -m cfut.remote {}'.format(sys.executable, workerid), additional_setup_lines=additional_setup_lines)
```

**Comment by sampsyo** [comment]
Indeed. Looks like this (and the problem) was introduced in #1.

**Comment by takluyver** [comment]
Closed by #5
